decisionengine/dijkstratesting/time_test.py

```diff
@@ -0,0 +1,213 @@
+                   'w': [('o',6), ('u',1), ('z',1)],
+                   'x': [('v',1), ('z',1)],
+                   'z': [('x',1), ('w',1)],
+                   'a0': [('a',3), ('b0',1), ('c0',3)],
+                   'b0': [('a0',1), ('c0',3), ('e0',2), ('i0',1)],
+                   'c0': [('a0',3), ('b0',1), ('d0',2)],
+                   'd0': [('c0',2), ('e0',1), ('f0',2), ('g0',2), ('h0',1)],
+                   'e0': [('d0',1), ('b0',2), ('h0',1)],
+                   'f0': [('d0',2), ('g0',3), ('p0',3)],
+                   'g0': [('d0',2), ('e0',1), ('p0',2), ('h0',2)],
+                   'h0': [('e0',1), ('d0',2), ('l0',2)],
+                   'i0': [('b0',1), ('j0',1), ('k0',3)],
+                   'j0': [('i0',1), ('l0',1)],
+                   'k0': [('i0',3), ('m0',1)],
+                   'l0': [('h0',2), ('j0',1), ('s0',1)],
+                   'm0': [('k0',1), ('n0',1)],
+                   'n0': [('m0',1), ('o0',2)],
+                   'o0': [('n0',2), ('w0',6)],
+                   'p0': [('f0',3), ('g0',1), ('q0',1)],
+                   'q0': [('r0',2), ('p0',1)],
+                   'r0': [('q0',1), ('v0',2)],
+                   's0': [('l0',1), ('t0',1)],
+                   't0': [('s0',1), ('v0',2), ('u0',1)],
+                   'u0': [('t0',1), ('w0',1)],
+                   'v0': [('t0',2), ('r0',2), ('x0',1)],
+                   'w0': [('o0',6), ('u0',1), ('z0',1)],
+                   'x0': [('v0',1), ('z0',1)],
+                   'z0': [('x0',1), ('w0',1)],
+                   'a1': [('a',2), ('b1',1), ('c1',3)],
+                   'b1': [('a1',1), ('c1',3), ('e1',2), ('i1',1)],
+                   'c1': [('a1',3), ('b1',1), ('d1',2)],
+                   'd1': [('c1',2), ('e1',1), ('f1',2), ('g1',2), ('h1',1)],
+                   'e1': [('d1',1), ('b1',2), ('h1',1)],
+                   'f1': [('d1',2), ('g1',3), ('p1',3)],
+                   'g1': [('d1',2), ('e1',1), ('p1',2), ('h1',2)],
+                   'h1': [('e1',1), ('d1',2), ('l1',2)],
+                   'i1': [('b1',1), ('j1',1), ('k1',3)],
+                   'j1': [('i1',1), ('l1',1)],
+                   'k1': [('i1',3), ('m1',1)],
+                   'l1': [('h1',2), ('j1',1), ('s1',1)],
+                   'm1': [('k1',1), ('n1',1)],
+                   'n1': [('m1',1), ('o1',2)],
+                   'o1': [('n1',2), ('w1',6)],
+                   'p1': [('f1',3), ('g1',1), ('q1',1)],
+                   'q1': [('r1',2), ('p1',1)],
+                   'r1': [('q1',1), ('v1',2)],
+                   's1': [('l1',1), ('t1',1)],
+                   't1': [('s1',1), ('v1',2), ('u1',1)],
+                   'u1': [('t1',1), ('w1',1)],
+                   'v1': [('t1',2), ('r1',2), ('x1',1)],
+                   'w1': [('o1',6), ('u1',1), ('z1',1)],
+                   'x1': [('v1',1), ('z1',1)],
+                   'z1': [('x1',1), ('w1',1)]
+                   }
+
+        self.g4 = {(1,1): [((1,2),1)],
```

Comment:
Make a comment on this variable too, and double check that all the other comments have a space after the hashtag then I'll approve it